src/resources/app/app.js

```diff
@@ -1,11 +1,12 @@
 import store from "./store/store";
 import Vue from "vue";
 import Vuetify from "vuetify/lib";
+import vuetify from "../../plugins/vuetify";
 import "vuetify/dist/vuetify.min.css";
 import { createInertiaApp } from "@inertiajs/inertia-vue";
 
 Vue.use(Vuetify);
-const vuetify = new Vuetify({});
+// const vuetify = new Vuetify({});
```

Comment:
will this be used?  Why don't just remove this line instead?

Reply:
sorry, it slipped. gonna fix it asap